test/org/opensolaris/opengrok/web/UtilTest.java

```diff
@@ -256,6 +258,19 @@ public void diffline() {
     }
 
     @Test
+    public void testEncode() {
+        String[][] tests = new String[][]{
```

Comment:
make sure all the if/else if cases in encode() are covered, e.g. `'\t'` is missing.